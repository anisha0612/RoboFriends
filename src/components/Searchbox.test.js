import React from "react";
import { create } from "react-test-renderer";
import SearchBox from "./Searchbox";

it("Check if the SearchBox component is rendered", () => {
  expect(create(<SearchBox />).toJSON()).toMatchSnapshot();
});
