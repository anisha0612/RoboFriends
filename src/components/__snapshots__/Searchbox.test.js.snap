// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check if the SearchBox component is rendered 1`] = `
<div>
  <input
    aria-label="Search Robots"
    className="ba bg-light-yellow mb5 pa3 br2"
    placeholder="Search Robots"
    type="search"
  />
</div>
`;
